api/config/20-api-deployment.yaml

```diff
@@ -20,6 +20,24 @@ spec:
           image: ko://github.com/tektoncd/hub/api/cmd/api
           ports:
             - containerPort: 8000
+          readinessProbe:
```

Comment:
Could you please add `probes` in DB deployment also?

Reply:
@SM43 that is unrelated to this patch perhaps another issue/bug.